spices/spices.mli

```diff
@@ -31,6 +32,7 @@ val reverse : bool -> style -> style
 val strikethrough : bool -> style -> style
 val underline : bool -> style -> style
 val width : int option -> style -> style
+val border : Border.t -> style -> style
```

Comment:
**suggestion**. from looking at how we're using this call (`border (get_border Thick)`) I think we could just export predefined borders from the Border module and use them like this, without needing a `get_border` function:

```ocaml
let overlay_border fmt =
  Spices.(default |> border Border.double |> build) fmt
```